mobile/src/main/java/org/openhab/habdroid/core/OpenHABVoiceService.java

```diff
@@ -111,9 +112,29 @@ public void onOpenHABTracked(String baseUrl, String message) {
         stopSelf(getLastStartId());
     }
 
+    /**
+     * @param message message to show
+     * @param messageType must be Constants.MESSAGES.DIALOG
+     * @param logLevel not implemented
+     */
+    public void showMessageToUser(String message, int messageType, int logLevel) {
+        switch (messageType) {
```

Comment:
if you only handle one possible value without any default, I would suggest using:
if (messageType != Constants.MESSAGES.DIALOG)
  return;